.travis.yml

```diff
@@ -14,30 +18,34 @@ matrix:
       dist: bionic
       before_install:
         - eval "$(curl -sL https://swiftenv.fuller.li/install.sh)"
+      before_script:
       script:
         - mv .Package.Test.swift Package.swift
         - swift build
         - swift test
     - env: JOB="POD_LINT"
-      osx_image: xcode12
+      osx_image: xcode12.5
       before_install:
       script:
         - pod lib lint
-    - env: JOB="XCODE" DEST="OS=14.0,name=iPhone 8" SCHEME="Swinject-iOS" SDK="iphonesimulator" ACTION="test" PLATFORM="iOS"
-      osx_image: xcode12
+    - env: JOB="XCODE" DEST="OS=14.0.1,name=iPhone 8" SCHEME="Swinject-iOS" SDK="iphonesimulator" ACTION="test" PLATFORM="iOS"
+      osx_image: xcode12.5
     - env: JOB="XCODE" DEST="arch=x86_64" SCHEME="Swinject-macOS" SDK="macosx" ACTION="test" PLATFORM="OSX"
-      osx_image: xcode12
+      osx_image: xcode12.5
     - env: JOB="XCODE" DEST="OS=14.0,name=Apple TV 4K" SCHEME="Swinject-tvOS" SDK="appletvsimulator" ACTION="test" PLATFORM="tvOS"
-      osx_image: xcode12
+      osx_image: xcode12.5
     - env: JOB="XCODE" DEST="OS=7.0,name=Apple Watch Series 6 - 44mm" SCHEME="Swinject-watchOS" SDK="watchsimulator" ACTION="build" PLATFORM="watchOS"
-      osx_image: xcode12
+      osx_image: xcode12.5
 
 before_install:
   - xcconfig=$(mktemp /tmp/static.xcconfig.XXXXXX)
   - trap 'rm -f "$xcconfig"' INT TERM HUP EXIT
   - echo 'EXCLUDED_ARCHS__EFFECTIVE_PLATFORM_SUFFIX_simulator__NATIVE_ARCH_64_BIT_x86_64__XCODE_1200 = arm64 arm64e armv7 armv7s armv6 armv8' >> $xcconfig
   - echo 'EXCLUDED_ARCHS = $(inherited) $(EXCLUDED_ARCHS__EFFECTIVE_PLATFORM_SUFFIX_$(EFFECTIVE_PLATFORM_SUFFIX)__NATIVE_ARCH_64_BIT_$(NATIVE_ARCH_64_BIT)__XCODE_$(XCODE_VERSION_MAJOR))' >> $xcconfig
   - export XCODE_XCCONFIG_FILE="$xcconfig"
+before_script:
+  - rm -r Swinject.xcodeproj
+  - xcodegen generate
```

Comment:
🚀

Reply:
Thanks @yoichitgy  ! 
I'll keep this PR with only this responsibility and I'll be opening another with a new CD workflow.